src/utils/unique-id.js

```diff
@@ -2,6 +2,6 @@ let index = 0;
 
 export default {
   generate() {
-    return 'mui-id-' + (index++);
+    return `mui-id-${index++}`;
```

Comment:
Looks like it's only used for TextField and enhanced-switch. I see two approaches:

1. Don't generate ids anymore by default, it's user's responsibility
2. https://www.npmjs.com/package/lodash.uniq